data_structures/src/data_request.rs

```diff
@@ -343,48 +343,50 @@ pub fn create_tally<RT>(
     dr_output: &DataRequestOutput,
     pkh: PublicKeyHash,
     report: &RadonReport<RT>,
-    reveals: Vec<RevealTransaction>,
-    n_commits: usize,
+    revealers: Vec<PublicKeyHash>,
+    committers: Vec<PublicKeyHash>,
 ) -> Result<TallyTransaction, failure::Error>
 where
     RT: TypeLike,
 {
     if let Stage::Tally(tally_metadata) = &report.metadata {
         let reveal_reward = dr_output.witness_reward;
+        let commits_count = committers.len();
+        let reveals_count = revealers.len();
+        let mut slashed_witnesses: HashSet<PublicKeyHash> = committers.iter().cloned().collect();
```

Comment:
Can't `committers` be cloned directly, provided `impl<T: Clone, S: Clone> Clone for HashSet<T, S>`?

Reply:
Ping on this, sorry for being picky ;)